### PR TITLE
Complete summarize company methods

### LINKED ISSUE #3: Complete `summarize_company_` methods in `Analyst` class
Complete helper methods within the `Analyst` class for summarizing data from respondents who completed the "Company" questions.

### PR DESCRIPTION
**Pull request by wengandrew**
Updates to `Analyst` class to summarize info from respondents who filled out the "Company" sections.

Includes some breaking changes to `Plotter`.

Plots are being generated by `explore_data.ipynb`

### CODE REVIEW

File: README.md
Comment (dzh-ad) [inline README.md:1]: Very thorough, and awesome ideas!

File: README.md [hunk 1]
```diff
@@ -11,27 +11,31 @@ Want to contribute? Do the following:
 1. `git clone` this repository to your local computer, or set up a GitHub
    codespace to the same effect.
 
-2. Set up your virtual environment and install the requirements. I'm running
+1. Set up your virtual environment and install the requirements. I'm running
    this on Python 3.13.1.
 
-  - Initialize the virtual environment:
-```
+Initialize the virtual environment:
```
Comment (dzh-ad) [inline README.md:17]: These commands are slightly different for Windows. Do we want to add instructions on how to create a virtual environment for different platforms, or trust that ppl can search/chatGPT their way through?

File: src/plotter.py [hunk 2]
```diff
@@ -161,18 +175,26 @@ def make_bar_plot_from_dictionary(self, this_dict,
         # Handle nans
         labels_new = ['nan' if x is np.nan else x for x in labels]
         labels_new = ['None' if x is None else x for x in labels_new]
+        labels_new = ['False' if x is False else x for x in labels_new]
+        labels_new = ['True' if x is True else x for x in labels_new]
 
         # Filter out excluded labels
```
Comment (dzh-ad) [inline src/plotter.py]: Copilot helped to simplify this snippet. See if it's any good?
```
        filtered_data = [(label, count) for label, count in zip(labels_new, counts) if label not in exclusions]
        # Check if there is any data left after filtering
        if filtered_data:
            labels_new, counts = zip(*filtered_data)
        else:
            labels_new, counts = [], []

        # Calculate the number of excluded elements
        exclusion_count = len(exclusions) - len(labels_new)

        # Recalculate the total items after exclusions
        total_items = sum(counts)
```

**Review by dzh-ad — COMMENTED** [review]
I didn't understand everything you've edited, but added some small comments

File: README.md (hunk 1 above)
Comment (wengandrew) [reply README.md:17]: Trust ppl for now; there won't be that many that stumble here.

**Pull request merged**